api/workAllocation2/index.ts

```diff
@@ -543,8 +543,7 @@ export async function getMyCases(req: EnhancedRequest, res: Response): Promise<R
       const mappedCases = checkedRoles ? mapCasesFromData(cases, checkedRoles as any) : [];
       result.total_records = mappedCases.length;
       result.unique_cases = getUniqueCasesCount(mappedCases);
-      const sortedCaseList = mappedCases.sort((a, b) => (a.isNew === b.isNew) ? 0 : a.isNew ? -1 : 1);
```

Comment:
@tarikdogan-hmcts  is this functionality common for both cases and access? Hence, cannot be removed. Please confirm

Reply:
agreed to merge as it is ,resolved